test/common/setInterval.spec.ts

```diff
@@ -1,6 +1,12 @@
 'use strict';
 import {zoneSymbol} from "../../lib/common/utils";
 
+var isNode = true;
```

Comment:
1) This is a better test `isNode = typeof window !== 'undefined'`.
2) could we add this to a separate file and import it from there?